onnxruntime/hosting/converter.h

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#pragma once
+
+#include <google/protobuf/stubs/status.h>
+
+#include "core/framework/data_types.h"
+
+#include "environment.h"
+#include "predict.pb.h"
+
+namespace onnxruntime {
+namespace hosting {
+
+onnx::TensorProto_DataType MLDataTypeToTensorProtoDataType(const onnxruntime::DataTypeImpl* cpp_type);
```

Comment:
Are you going to address this comment?

Reply:
From the implementation the type of `cpp_type` could not be MLDataType. Otherwise the `onnxruntime::DataTypeImpl::GetType` will not work. I'd like to keep it as-is unless there is a different way to implement it.